app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -196,33 +196,55 @@ class _ChatInput extends ConsumerStatefulWidget {
 }
 
 class __ChatInputState extends ConsumerState<_ChatInput> {
-  TextEditingController controller = TextEditingController();
+  late ActerTextController textController;
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    _setController();
+  }
+
+  void _setController() {
+    final Map<String, TextStyle> triggerStyles = {
+      '@': TextStyle(
+        color: Theme.of(context).colorScheme.onSecondary,
+        height: 0.5,
+        background: Paint()
+          ..color = Theme.of(context).colorScheme.secondary
+          ..strokeWidth = 10
+          ..strokeJoin = StrokeJoin.round
+          ..style = PaintingStyle.stroke,
+      ),
+    };
+    textController = ActerTextController(triggerStyles: triggerStyles);
```

Comment:
why not do that in the class directly?

Reply:
That'd mean we are taking away the customization support from the plugin, which is what it needs to offer...